test/data/audio-tracks.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
```

Comment:
I don't know if it matters, but this JSON is invalid due to not double quoting all keys and non-number/boolean values